salesforce/aa-lwc/package.json

```diff
@@ -24,5 +24,30 @@
   },
   "dependencies": {
     "jsonwebtoken": "^9.0.2"
+  },
+  "overrides": {
+    "@lwc/template-compiler": {
```

Comment:
Overrides were made at the lowest possible level -- for example here @babel/traverse is replaced only for @lwc/template-compiler and its dependencies (unless a sub dependency specifies another version/range).